eventsourcing/src/main/java/org/axonframework/eventsourcing/AbstractSnapshotter.java

```diff
@@ -92,10 +97,23 @@ private void doScheduleSnapshot(Class<?> aggregateType, String aggregateIdentifi
             }
         }
         if (snapshotsInProgress.add(typeAndId)) {
+            String rootTraceName = String.format("createSnapshot %s",
```

Comment:
Why has the notation changed from dot-separation to space-separation? 
Or is the intent to use the space notation? 

At any note, I think we should choose one or the other and not a mix.
I'd anticipate request from users about that in the future otherwise, which me might not be able to change as people expect the naming once we release this.